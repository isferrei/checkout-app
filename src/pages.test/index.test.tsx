import { render } from "@testing-library/react";
import Home from "../pages/index";

describe("Index Page", () => {
  test("renders the page content correctly", () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
