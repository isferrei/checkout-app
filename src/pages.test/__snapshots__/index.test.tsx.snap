// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders the page content correctly 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center items-center h-screen"
  >
    <h2>
      Failed to fetch data. Please try again later.
    </h2>
  </div>
</DocumentFragment>
`;
